spec/defines/error_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'clickhouse::error' do
+  let(:title) { 'Error message' }
+  let(:params) do
+    {}
+  end
+
+  it { is_expected.to compile }
+  it do
+    is_expected.to contain_exec('Error message')
+      .with_command('/$')
```

Comment:
I don't understand the purpose of the string `/$` can you please elaborate ? If it is a placeholder does it become a problem when you make all strings immutable here with `# frozen_string_literal: true`.